k8s/ace-rp/Makefile

```diff
@@ -77,6 +77,9 @@ set-images: kustomize
 
 .PHONY: deploy-ace-rp
 deploy-ace-rp: prechecks kustomize kubectl
+	@docker save $(ACE_RP_IMG) > ace-rp.image
+	@minikube image load ace-rp.image
```

Comment:
@soluchok @soluchok On a second note, these makefiles are used by downstream repos to deploy the components on cloud k8s, which wouldn't have minikube, so it'll fail.

Can we add these commands in [sanbox-makefile](https://github.com/trustbloc/sandbox/blob/main/k8s/Makefile) or fix existing [rm/load commands](https://github.com/trustbloc/sandbox/blob/main/k8s/Makefile#L42-L54) ?

Reply:
This has been updated in https://github.com/trustbloc/sandbox/pull/1076#pullrequestreview-678941850.